src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1991,9 +2010,12 @@ def create_system_user(self, userName: str, zoneName: str):
         except SubprocessError as error:
             raise DashboardException(error, http_status_code=500, component='rgw')
 
-    def get_user_list(self, zoneName: str):
+    def get_user_list(self, zoneName=None, realmName=None):
```

Comment:
we used to follow `snake_case` in python for variable names and especially API params right?

Reply:
Yes, but since zoneName was camelCase so kept realmName the same. Will take care in future PRs.